hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java

```diff
@@ -195,6 +195,15 @@ public String getHostName() {
     return hostName;
   }
 
+  /**
+   * Returns the Hostname and IP of the datanode separated by a slash.
+   *
+   * Eg: datanode001.corp/192.168.0.123
+   */
+  public String getHostNameAndIP() {
+    return getHostName() + "/" + getIpAddress();
```

Comment:
I would love to make this (or UUID + this) the default toString, and renaming the existing more verbose toString to something like `toDebugString`.  This way we could avoid having to convert manually in all places, and use the verbose one explicitly if needed.

Reply:
I'm fine with adding UUID into the string too. It would solve the problem with mini-cluster where all the nodes are on the same host / IP. I wonder if anything would break if we changed the default datanode details to string? Almost everywhere it is logged, we don't call about all the verbose info it currently includes. We really just want to the host / IP / ID.